src/tutorials/cloudery/index.md

```diff
@@ -0,0 +1,67 @@
+# Cloudery
+
+This documentation explain how to use our Cloudery API to create and manager our own Cozy.
+Code example is provided for Ruby language, but this can be easily ported to any other langage, there is no real dependency to the language by itself.
+
+Full API documentation is provided [here](https://cloudery.cozycloud.cc/api/) and this documentation is only about some standard usages examples.
+
+## Creating a Cozy
+
+### From web
+
+Send your user on `/v2/{partner}/onboard`. You can customize some onboarding parameters with URL query, see [the documentation](https://cloudery.cozycloud.cc/api/#tag/instances/paths/~1{partner}~1onboard/get).
+Our Cloudery will onboard the user throught differents steps to gather required informations and then create the instance.
+
+![email](email.png)
+![FQDN](fqdn.png)
+![terms](terms.png)
+![confirm](confirm.png)
+
+Your user will received an email at the end, with an activation link to click to go on the initial passphrase definition.
+
+![validate](validate.png)
+![passphrase](passphrase.png)
+
+### From API
```

Comment:
```suggestion
### From API called by a client app
```

The idea behind this change, is to separate the call from server side and call from client side. Since Client side has to use the `/public/` API.

Reply:
:ok_hand: